src/components/DateInput/index.tsx

```diff
@@ -36,13 +36,11 @@ const StyledWrapper = styled.div`
   }
   .calendar {
     font-family: 'Asap';
-    border-color: ${({ theme: { colorsDeprecated } }) =>
-      colorsDeprecated.gray300};
+    border-color: ${({ theme }) => theme.colors.neutral.border};
```

Comment:
```suggestion
    border-color: ${({ theme }) => theme.colors.neutral.borderWeak};
```

As background is light (weak) we should use border in this context